be/src/exec/vectorized/hdfs_scanner.cpp

```diff
@@ -213,7 +213,16 @@ void HdfsScannerContext::set_columns_from_file(const std::unordered_set<std::str
     }
 }
 
-void HdfsScannerContext::append_not_exised_columns_to_chunk(vectorized::ChunkPtr* chunk, size_t row_count) {
+void HdfsScannerContext::update_not_existed_columns_of_chunk(vectorized::ChunkPtr* chunk, size_t row_count) {
+    if (not_existed_slots.empty() || row_count <= 0) return;
+
+    ChunkPtr& ck = (*chunk);
+    for (auto* slot_desc : not_existed_slots) {
+        ck->get_column_by_slot_id(slot_desc->id())->append_default(row_count);
+    }
+}
```

Comment:
I want to know why not set num_row to Chunk

Reply:
I think the caller should not depends on this method to adjusting the actual row size of `chunk`